src/test/java/com/johncnstn/auth/constraints/PasswordValidatorTest.java

```diff
@@ -0,0 +1,57 @@
+package com.johncnstn.auth.constraints;
+
+import org.junit.jupiter.api.Test;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+
+public class PasswordValidatorTest {
+
+    @Test
+    public void testNull() {
+        testPassword(null, false);
+    }
+
+    @Test
+    public void testEmpty() {
+        testPassword("", false);
+    }
+
+    @Test
+    public void testWithWhitespace() {
+        testPassword("Password 1", false);
```

Comment:
For fake data use java faker.
View utils class for this purposes.

Reply:
Okay
Could you please point me to a faker method which will create a string which is a valid password with the only exception that it contains a whitespace